app/views/editor/thang/ThangTypeEditView.coffee

```diff
@@ -378,7 +378,7 @@ module.exports = class ThangTypeEditView extends RootView
 
         @fileLoaded()
       else if (data.error)
-        noty({ text: "Error occurred. Check console. Please inform eng team and provide Adobe Animate File.", type:"error", timeout: 10000})
+        noty({ text: "Error occurred. Check console. Please inform eng team and provide Adobe Animate File.", type:"error", timeout: 10000000 })
```

Comment:
Why the huge timeout? (Non blocking)

Reply:
The intention here was to make the notification sticky because the imports can take a significant amount of time.  I assume the imports will start and then people will sometimes switch to another tab / start something else.  In the event that an error occurs I want to be sure that the user is aware and doesn't miss the timeout.  I looked at the API and switched it to be a sticky notification - good call out.